src/Frontend/Page/Delegation.hs

```diff
@@ -181,22 +182,18 @@ viewDelegationNetwork (fromMaybe (DScopeIdeaSpace SchoolSpace) -> scope) = do
 
 delegationInfos :: DScope -> EQuery DelegationNetwork
 delegationInfos scope = do
-    delegations <- findImplicitDelegationsByScope scope
+    delegations <- Map.toList <$> findImplicitDelegationsByScope scope
 
     -- Create delegations
-    let mkGraphNode (de, _, dees) = (unDelegate de, unDelegate de, unDelegatee <$> dees)
+    let mkGraphNode (de, dees) = (unDelegate de, unDelegate de, unDelegatee . snd <$> dees)
 
     -- Build graphs and graph handler functions
     let graphNodes = fixLeaves $ mkGraphNode <$> delegations
     let (delegationGraph, _vertexToGraphNode, nodeToVertex) = graphFromEdges graphNodes
     let uidToVertex = fromJust . nodeToVertex
     let graphComponents = stronglyConnComp graphNodes
 
-    -- Count number of inbound delegation edges in local 'DScope'.  This is not the 'votingPower'
-    -- because it does not take into account delegations from surrounding 'DScope's, but it is much
-    -- cheaper to calculate.  (FUTUREWORK: it would be nice to have the actual voting power here,
-    -- but that should go together with showing the implicit (inherited) delegation edges in the
-    -- graph as well, e.g. as dashed lines.)
+    -- Count number of inbound delegation edges in implicit 'DScope'.
```

Comment:
An implicit `DScope` could be an ancestor to the local one.  I think what you want to say here is

```haskell
    -- Count voting power (number of inbound delegation edges in the local 'DScope' and all its ancestors).
```